src/main/java/games/strategy/engine/config/GameEngineProperty.java

```diff
@@ -1,13 +1,16 @@
 package games.strategy.engine.config;
 
 public enum GameEngineProperty {
-  MAP_LISTING_SOURCE_FILE("Map_List_File"), ENGINE_VERSION("engine_version"), LOBBY_PROPS_URL(
-      "lobby_properties_file_url"), LOBBY_PROPS_BACKUP_FILE("lobby_properties_file_backup");
-
+  MAP_LISTING_SOURCE_FILE("Map_List_File"),
+  ENGINE_VERSION("engine_version"),
+  LOBBY_PROPS_URL("lobby_properties_file_url"),
+  LOBBY_PROPS_BACKUP_FILE("lobby_properties_file_backup"),
+  MAP_FOLDER("map_folder"),
+  SAVED_GAMES_FOLDER("saved_games_folder");
 
   private final String value;
 
-  private GameEngineProperty(final String value) {
+  GameEngineProperty(final String value) {
```

Comment:
Was the accessibility increased for test purposes?  `@VisibleForTesting`?  However, I don't see any usage outside this class.